### ISSUE TITLE
[Supporting the platform – Make Code - Extensions] Focus indicator is not visible on controls in high contrast theme under 'Extensions' screen. 

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/pxt-microbit#3690]
**User Experience**:
User will find difficulty to see any control if keyboard focus is not visible on any interactive control in high contrast theme as they won't be able to take an action on such control and also they will surely miss the functionality and will proceed further.

**Note**: User credentials should NOT be included in the bug.

**Pre-Requisite:**
Turn on Windows High contrast theme (Settings->Ease of access->High Contrast->Turn on High Contrast->Select High Contrast White and Black

**Repro Steps:**
1. Launch application URL.
2. Home page will open.
3. Navigate "New Project" option and select it.
4. Editor screen will open.
5. Navigate to "Settings" option appearing in header section and select it.
6. Navigate to "Extensions" option and select it.
7. Navigate through all the controls appearing under "Extensions" page
8. Verify focus visibility on controls like (Bluetooth services, Devices, Radio-broadcast, Servo etc.) after applying high contrast theme(Black/White).

**Actual Result:**
Keyboard focus is not visible on controls like (Bluetooth services, Devices, Radio-broadcast, Servo etc.) link after applying high contrast theme(Black/White) under 'Extensions' screen and due to this visual impaired user is unable to see the keyboard focus on that control and also he/she is unable to perform functionality over such control.

**Expected Result:**
Keyboard focus should be visible on controls like (Bluetooth services, Devices, Radio-broadcast, Servo etc.) link after applying high contrast theme(Black/White) under 'Extensions' screen, so that visual impaired user could easily see the keyboard focus on that control to perform functionality over such control.

**WCAG Reference:**
[WCAG 4.3.1 - No Disruption of Accessibility Features
](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}})

**Test Environments:**
Version: 20H2 (OS Build: 19042.508)
Browser: Edge chromium DEV Build (Version 87.0.654.0)

**Comment by Shalini3457** [comment]
Labels:
#Accessibility; #A11ySev2; #HCL; #Benchmark-HCL-MakeCode-OCT2020; #WCAG4.3.1; #Win10-Edge(Chromium)

[WCAG4.3.1_Extenions_HC.zip](https://github.com/microsoft/pxt-microbit/files/5535064/WCAG4.3.1_Extenions_HC.zip)

**Comment by shakao** [comment]
fixed with https://github.com/microsoft/pxt/pull/8063
